testing/regression/cypress/e2e/features/page-library-sort.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User can view existing page library data here.
+
+    Background:
+        Given I am logged in as "superuser" and password ""
+
+    Scenario: User list Page name in descending and ascending order
+        Given User navigates to Page Library
+        When User views the Page library
```

Comment:
Question: Whats the purpose of having the navigate step and then the view step? 

The view step calls `cy.get('#pageLibrary');`.  Is the purpose to allow time for the page to load prior to executing the next step? If so, perhaps it should just be added to the navigation step.

Reply:
Even after navigation, I am checking weather redirected to the correct page-library or not.